pulser/simulation/simconfig.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2020 Pulser Development Team
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import numpy as np
+import qutip
+from typing import List
+
+noise_dict_set = {'doppler', 'amplitude', 'SPAM', 'dephasing'}
+
+
+class SimConfig:
+    """Include additional parameters to simulation. Will be necessary for
+        noisy simulations.
+
+        Keyword arguments:
+            noise_types (string list): types of noises to be used in the
+                simulation.
+            samples_per_run (int value): number of samples per noisy run.
+                Useful for cutting down on computing time, but unrealistic.
+            runs (int value): number of runs needed : each run
+                draws a new random noise.
+            laser_waist (float): waist of the gaussian laser in global pulses.
+            solver_options (qutip.Options): options for the qutip solver.
+        """
+    def __init__(self, noise_types: List[str] = [], runs: int = 15,
+                 samples_per_run: int = 5, temperature: float = 50.,
+                 laser_waist: float = 175.,
+                 solver_options: qutip.Options = qutip.Options(max_step=5)):
+        self.noise = noise_types
+        self.temperature = temperature
+        self.runs = runs
+        self.samples_per_run = samples_per_run
+        self.laser_waist = laser_waist
+        self.solver_options = solver_options
```

Comment:
This type of construction is quite suitable for a `dataclass`, I think it would be more elegant to use it. We use it already on `Device` and `Channel`, you should take a look